tasks/approve.yml

```
@@ -1,8 +1,19 @@
 ---
+- name: Get total number of current nodes
+  # skip_yaml_lint
+  ansible.builtin.command: |
+    kubectl get nodes --no-headers \
+      --kubeconfig {{ openshift_kubeconfig_file }} | wc -l
+  register: csr_autoapprove_current_nodes
+  changed_when: true
```

```suggestion
  changed_when: false
```
Vorschlag: changed_when auf false, weil nichts geändert wird sondern nur infos ausgelesen werden

---

find ich gut. hab ich eingebaut.